w3lib/url.py

```diff
@@ -73,7 +76,7 @@ def safe_url_string(url, encoding='utf8', path_encoding='utf8'):
         to_native_str(netloc).rstrip(':'),
 
         # default encoding for path component SHOULD be UTF-8
-        quote(to_bytes(parts.path, path_encoding), _safe_chars),
+        quote(to_bytes(parts.path, path_encoding), _safe_chars) if quote_path else to_native_str(parts.path),
```

Comment:
Any change we can refactor things a bit to keep this like shorter?

Reply:
The only way I found was to write if ans else in seperate lines to make it look shorter.

Reply:
I don’t think that would be a bad idea. Using a regular if/else statement before the return statement, and using here the corresponding value.

Reply:
@Gallaecio is this fine?